spec/rep_template_spec.rb

```diff
@@ -106,5 +106,18 @@
         end.to raise_error(/The max_containers prop should be a positive integer/)
       end
     end
-  end  
+  end
+
+  describe 'volume_mounted_files.erb' do
+    let(:template) { job.template('bin/volume_mounted_files') }
+
+    context 'checks the max_containers value' do
+      it 'raises an error if max_containers is <= 0' do
+        deployment_manifest_fragment['diego']['rep']['max_containers'] = -10
+        expect do
+          rendered_template
+        end.to raise_error(/The max_containers prop should be a positive integer/)
+      end
+    end
+  end
```

Comment:
I don't think that this is the test you meant to be here 😄

Reply:
Hello, could you elaborate on that comment? I want to ensure I understand what you see as incorrect about the test placement and what you recommend changing.

Reply:
Initially I thought you accidentally copy and pasted it from the test above on accident and forgot to fill it out. Now that I look closer, I see that you copy and pasted it _on purpose_ from the test above.

😅 My mistake!